documentation/content/en/books/fdp-primer/writing-style/_index.adoc

```diff
@@ -321,7 +352,7 @@ This rule proposes better wording.
 
 . FreeBSD.Hyphens: Often adverbs ending with 'ly' are added with a hyphen which is wrong.
 
-. FreeBSD.Spacing: Often double spaces are hard to catch with the naked eye and this is addressed here.
```

Comment:
Was the presence of a listing for this rule consistent with the `NO` below? 

https://github.com/freebsd/freebsd-doc/blob/ac4fd34edfa1e5e2edb6fb9fc61acd782a0ed33b/.vale.ini#L27